### PR TITLE
add logic

### PR DESCRIPTION
**Pull request by yahyaelganyni1**
I added a logic directory with a button and calculate files that handle the logic of the calculator then implement the function in the calculator file

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,33 +1,185 @@
 /* eslint-disable */
 import React, { Component } from 'react';
 import Button from './Button';
+import calculate from '../logic/calculate';
```
Comment (DeliceLydia) [inline src/components/Calculator.js]: - [ ] The `Calculator` component should import the `calculate.js` & `operate.js` files from the logic directory currently you are importing `calculate` only kindly fix that

Comment (DeliceLydia) [reply src/components/Calculator.js]: # Optional

- [x] Currently your design does not look like the given one you can fix that
Design
design

yours
ds

you can improve it a little bit

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @yahyaelganyni1👏,

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (yahyaelganyni1) [reply src/components/Calculator.js]: hello Lydia  
I can't find a requirement for importing `thecalculate.js` & `operate.js  ` to the `Calculator ` component

Untitled

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉 🎉 

### Hi @yahyaelganyni1 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**